src/Microsoft.DotNet.Interactive.Documents.Tests/CodeSubmissionFormatTests.dib_file_can_be_round_tripped_through_read_and_write_without_the_content_changing.approved.dib

```diff
@@ -0,0 +1,27 @@
+#!meta
+
+{"kernelInfo":{"defaultKernelName":"csharp","items":[{"aliases":["cs","C#","c#"],"name":"csharp"},{"aliases":["fs","F#","f#"],"name":"fsharp"},{"aliases":["powershell"],"name":"pwsh"},{"name":"mermaid"},{"name":"javascript"}]}}
```

Comment:
Can we format/indent this?  It's kind-of meaningful to human eyes and if/when this changes, the diff will be more manageable.

Reply:
Yes it should work if formatted over multiple lines, but I need to add a test for that.